src/Cli/func/Common/Utilities.cs

```diff
@@ -293,5 +293,46 @@ internal static bool IsMinifiedVersion()
 
             return false;
         }
+
+        internal static string ExtractIKeyFromConnectionString(string connectionString)
```

Comment:
Can we rename it to `ExtractInstrumentationKeyFromConnectionString`

Reply:
I started with that name, but it felt too long. 'IKey' is a common and well-known term for instrumentation key.

Reply:
Same comment as below, I don't mind the naming being long as long as it makes sense and is descriptive - not blocking though!